setup.py

```diff
@@ -26,13 +29,23 @@
     data_files.append(("/etc", ['config/pylintrc']))
 
 
+def get_git_version():
+    if os.path.exists(PKG_INFO):
+        with open(PKG_INFO) as package_info:
+            for key, value in (line.split(':', 1) for line in package_info):
+                if key.startswith('Version'):
+                    return value.strip()
+
+    return subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
+
+
 setup(
     name='strato_pylint',
 
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.0.0',
+    version=get_git_version(),
```

Comment:
I think so, it's installed in the base-image and I'm not sure it's installed on the slaves/build_chroot